.github/workflows/pr.yml

```diff
@@ -1,5 +1,6 @@
 name: PR Build
 on:
+  workflow_dispatch:
```

Comment:
What is this for?

Reply:
this lets you run the ci through the actions tab